### ISSUE TITLE
[4.x]: UK counties no longer available

### ISSUE THREAD

**Issue by chrismou** [craftcms/cms#13101]
### Description

After upgrading to Craft Commerce 4, and spending far too long debugging to try to find out what Ive missed, it seems that the new Administrative Area concept doesn't include British Counties.

The issue is mostly around shipping in Commerce. The UK incorporates England, Scotland Wales and Northern Ireland, and while we're effectively considered a single country in Craft, in reality Northern Ireland is on the other side of the Irish Sea, and the Scottish Highlands are well outside of most shipping companies scope for, say, one day delivery.

So up to now we've based shipping rules on counties. But since upgrading, its no longer possible. This massively complicates things as, using DPD as an example, this is what they consider the Scottish Highlands

I tried 

### Steps to reproduce

1.

### Craft CMS version

4.x

### PHP version

8.1

### Installed plugins and versions

-

**Comment by i-just** [comment]
Hi, thanks for reaching out. It sounds like you might want to have a look at: https://github.com/craftcms/cms/discussions/11788. 

The library ([commerceguys/addressing](https://github.com/commerceguys/addressing)) doesn’t come with UK subdivisions (more details in that discussion), but the solution mentioned there should allow the input and saving of plain text values.

You might also find this helpful: https://github.com/craftcms/cms/issues/12644.

**Comment by chrismou** [comment]
@i-just Thanks, that actually is really useful. I've already switched the shipping rules to be postcode based, but I'd not had a chance to look into the counties not saving so hopefully this will solve it for me :+1:

**Comment by bymayo** [comment]
@i-just Just jumping in on this as I posted a duplicate (Sorry!) But I don't see the solution Brandon posted actually allowing to display the Counties for UK, only display a field (Which i'm assuming as it's untested, just displays an input field, not a select)

I'm also a little confused as to why this can't be added to the core? As I assume a lot of developers will be using Craft and Craft Commerce in the UK, and to neglect having UK Counties in the system and needing to add them manually with a module seems a bit crazy to me.

**Comment by i-just** [comment]
Yes, if you use those events, the `administrativeArea` field will be a text input in the Control Panel, but you can render it as a dropdown in the front end. Additionally, from `4.5`, you’ll be able to override the default address formatter and display the `administrativeArea` as part of the card view (https://github.com/craftcms/cms/pull/13242).

Data for the library ([commerceguys/addressing](https://github.com/commerceguys/addressing)) used to come from Google (issue explaining why counties are not provided: https://github.com/google/libaddressinput/issues/193). It looks like `commerceguys/addressing` has taken ownership of that: https://github.com/commerceguys/addressing/blob/master/README.md?plain=1#L19, but as with Google’s dataset, it doesn’t come with UK subdivisions.

**Comment by brandonkelly** [comment]
Craft 4.5.0 is out with a new event that makes it possible to customize address subdivisions (#13361).
